rudder-docker.yml

```diff
@@ -27,6 +27,10 @@ services:
       - CONFIG_BACKEND_URL=https://api.rudderlabs.com
       - WORKSPACE_TOKEN=<your_workspace_token>
       - CONFIG_PATH=/app/config/config.toml
+      # - RSERVER_BACKEND_CONFIG_CONFIG_FROM_FILE=true
+    # Uncomment the following lines to mount workspaceConfig file
+    # volumes:
+    #   - <absolute_path_to_workspace_config>:/etc/rudderstack/workspaceConfig.json
```

Comment:
I am guessing this is the default value in config.toml for the workspace config file path. Instead of assuming that, maybe we can redefine that variable here uring RSERVER_ format? I am approving either way. Feel free to ignore this if you think it's not important.

Reply:
Done